src/SFA.DAS.Data.Functions/Ioc/DefaultRegistry.cs

```diff
@@ -0,0 +1,68 @@
+﻿using MediatR;
+using Microsoft.Azure;
+using SFA.DAS.Configuration;
+using SFA.DAS.Configuration.AzureTableStorage;
+using SFA.DAS.Data.Application.Configuration;
+using SFA.DAS.NLog.Logger;
+using StructureMap;
+
+namespace SFA.DAS.Data.Functions.Ioc
+{
+    public class DefaultRegistry : Registry
+    {
+        private string ServiceName = CloudConfigurationManager.GetSetting("ServiceName");
+        private const string Version = "1.0";
+
+        public DefaultRegistry()
+        {
+            Scan(scan =>
+            {
+                scan.AssembliesFromApplicationBaseDirectory(a => a.GetName().Name.StartsWith("SFA.DAS."));
+                scan.RegisterConcreteTypesAgainstTheFirstInterface();
+            });
+
+            var config = GetConfiguration();
+
+            For<IDataConfiguration>().Use(config);
+            RegisterRepositories(config.DatabaseConnectionString);
+            AddMediatrRegistrations();
+
+            ConfigureLogging();
+        }
+
+
+        private void RegisterRepositories(string connectionString)
```

Comment:
Empty function can be removed unless its worth leaving as will have code when function is implemented?